libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -400,13 +400,17 @@ void AP_BLHeli::msp_process_command(void)
         msp_send_reply(msp.cmdMSP, (const uint8_t *)UDID_START, 12);
         break;
 
+        // a literal "4" is used for the PWMType here to allow Rover
+        // to use the same number for the same protocol.  At time of
+        // writing the AP_MotorsUGV::PWMType has not been unified with
+        // AP_Motors::PWMType.
     case MSP_ADVANCED_CONFIG: {
         debug("MSP_ADVANCED_CONFIG");
         uint8_t buf[10];
         buf[0] = 1; // gyro sync denom
         buf[1] = 4; // pid process denom
         buf[2] = 0; // use unsynced pwm
-        buf[3] = (uint8_t)PWM_TYPE_DSHOT150; // motor PWM protocol
+        buf[3] = 4; // (uint8_t)AP_Motors::PWMType::DSHOT150;
```

Comment:
Surely this is defined by MSP?

Reply:
I think its an extension of some description...

inav defines it as 4 here: https://github.com/iNavFlight/inav/blob/master/src/main/drivers/pwm_mapping.h#L44

betaflight defines it as 5 here: https://github.com/betaflight/betaflight/blob/master/src/main/drivers/motor.h#L35

... and those packets are significantly different.

So.  Yeah.  I could add `#define` here, but I don't think it really gains us that much.  I'm really just after sending the same number here for all ArduPilot vehicles - we may as well be consistent with ourselves, even if we can't be consistent with both inav and betaflight!